memtier_benchmark.cpp

```diff
@@ -90,6 +90,7 @@ static void config_print(FILE *file, struct benchmark_config *cfg)
         "clients = %u\n"
         "threads = %u\n"
         "cpu_split = %s\n"
+        "compress_perc = %d\n"
```

Comment:
consider calling this param compression_ratio.

Reply:
btw, should it be `%u`?

Reply:
yes it should